### DISCUSSION TITLE
UI Revamp: asking for feedback during the UI implementation

### DISCUSSION THREAD

**Discussion by thegostev** [ckan/ckan#8399] [CKAN 3.0]
Hi,

At the start of the UI Revamp project with the goal of producing the new CKAN UI (, design and design system), it was decided to work in the open. It happened like this during the design stage in [Figma](https://www.figma.com/design/oI3wDSdZ0cHZLh6CrfYMnb/CKAN-3.0-Design-System?node-id=546-2692&t=mQFXuWOIgulMcsGt-0), worked well in terms of gathering feedback and incorporating it in the work. Thus it look reasonable to proceed working in the open during the implementation.

The UI works currently happening in this instance specifically created for the progress demonstration purposes: 
https://ckan-design-preview.galv1.links.com.au/en/

login: ckan
password: preview

Please share your feedback and ideas as the implementation moves on.

**Comment by amercader** [comment]
It looks really nice, great work @aleeexgreeen , Petar and the rest of the team.

I'd probably populate it with more datasets to help get a good first impression, and add a small banner or notice somewhere stating this is a WIP, maybe linking to a place where people can give feedback (a github discussion is fine).
